roles/confluent.kafka_broker/tasks/set_principal.yml

```diff
@@ -33,6 +33,7 @@
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['sasl_protocol'] | default(sasl_protocol) | normalize_sasl_protocol == 'none'
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['ssl_enabled'] | default(ssl_enabled) | bool
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['ssl_mutual_auth_enabled'] | default(ssl_mutual_auth_enabled) | bool
+  diff: "{{ not mask_diff|bool }}"
```

Comment:
should this no_log? there is a secret in the shell command

Reply:
Agree. Updated the code.